buildlib/dockers/ubuntu-release.Dockerfile

```diff
@@ -1,10 +1,12 @@
-ARG CUDA_VERSION=10.1
-ARG UBUNTU_VERSION=16.04
+ARG CUDA_VERSION
+ARG UBUNTU_VERSION
 FROM nvidia/cuda:${CUDA_VERSION}-devel-ubuntu${UBUNTU_VERSION}
 
+ARG DRIVER_VERSION
```

Comment:
Maybe rename to NVIDIA_DRIVER_VERSION?

Reply:
This would make it less readable in the docker-compose.yml file.